### ISSUE TITLE
PaymentIntent create method returns null

### ISSUE THREAD

**Issue by crismarksotto** [luigel/laravel-paymongo#1]
I used the code below to test the payment intent but it returns null when I used dd($paymentIntent); 

PaymentMethod is working but the paymentintent create method is not working on my end.

```
$paymentIntent = Paymongo::paymentIntent()->create([
            'amount' => 10000,
            'payment_method_allowed' => [
                'card'
            ],
            'payment_method_options' => [
                'card' => [
                    'request_three_d_secure' => 'automatic'
                ]
            ],
            'description' => 'This is a test payment intent',
            'statement_descriptor' => 'LUIGEL STORE',
            'currency' => "PHP"
        ]);

dd($paymentIntent); 
```

Note:
- Paymongo api key is already in .env
- I'm using api key test
- Im using Laravel 7
- Paymongo Account is activated

**Comment by luigel** [comment]
I can't replicate your problem.

![image](https://user-images.githubusercontent.com/26235960/79832904-f5e9b780-83dc-11ea-84c3-9cd2a83e1fcf.png)

I'm using laravel-paymongo v1.1.1.
Here is my code

``` php
Route::get('test-payment', function () {
    $paymentIntent = Paymongo::paymentIntent()->create([
        'amount' => 100,
        'payment_method_allowed' => [
            'card'
        ],
        'payment_method_options' => [
            'card' => [
                'request_three_d_secure' => 'automatic'
            ]
        ],
        'description' => 'This is a test payment intent',
        'statement_descriptor' => 'LUIGEL STORE',
        'currency' => "PHP",
    ]);

    dd($paymentIntent);
});
```

**Comment by crismarksotto** [comment]
Thanks Problem solved! Downgraded to v1.1.1
